R/plotPrevalence.R

```diff
@@ -485,7 +486,7 @@ setMethod("plotTaxaPrevalence", signature = c(x = "SummarizedExperiment"),
                                     size_by = NULL,
                                     alpha = line_alpha,
                                     linetype = line_type,
-                                    size = line_size)
+                                    linewidth = line_size)
```

Comment:
Why the name has changed?

Reply:
size should be replaced with linewidth in ggplot2 3.4.0